cmd/infracost-generate-docs/main.go

```diff
@@ -0,0 +1,128 @@
+package main
```

Comment:
how do you feel about having this in scripts/docs-generator as that folder has other helpers? I was thinking cmd might be used for product subcommands (e.g. `infracost diff`) but I'm not sure yet. 
Also, what do you think about updating the Makefile so we can have `make docs`?

Reply:
Since this script is an acutal golang command line app, I prefered to put it in `cmd/`, on the other hand `infracost diff`  and other subcommands' logic should reside in the `infracost` cmd since the subcommand is just an arg.
https://github.com/influxdata/influxdb/tree/master/cmd is an example of using multiple cmds
Anyway, there is no silverbullet so we just need to have a standard, what you think?
Regarding the makefile I agree with you.

Reply:
you're right, cmd makes more sense for this.